examples/acoustic/Acoustic_codegen.py

```diff
@@ -29,16 +29,13 @@ def __init__(self, model, data, source, nbpml=40, t_order=2, s_order=2,
         self.model.nbpml = nbpml
         self.model.set_origin(nbpml)
 
-        def damp_boundary(damp):
+        def damp_boundary(damp, nbp):
```

Comment:
A small docstring or comment explaining what `nbp` is? Otherwise this looks quite cryptic.